types/src/v0/traits.rs

```diff
@@ -498,6 +503,7 @@ pub trait SequencerPersistence: Sized + Send + Sync + 'static {
         // starting in a view in which we had already voted before the restart, and prevents
         // unnecessary catchup from starting in a view earlier than the anchor leaf.
         let view = max(highest_voted_view, leaf.view_number());
+        let epoch = todo!();
```

Comment:
No idea where this should come from. I'd expect there to be `Leaf2::epoch_number()`, but that's not a thing.